deploy/kubectl/hostpath-operator.yaml

```diff
@@ -90,7 +73,7 @@ spec:
       containers:
       - name: openebs-provisioner-hostpath
         imagePullPolicy: IfNotPresent
-        image: openebs/provisioner-localpv-ci:dev-120403
```

Comment:
this tag is used by E2e scripts for replacing with the required release tag. ths name also might also have a dependency. ( cc: @nsathyaseelan )

Reply:
Yes @kmova we were replacing this name in e2e tests. will modify the test case.

Reply:
@nsathyaseelan @kmova -- Should I revert all changes for this file?

Reply:
No @niladrih you don't revert the changes. I will update the test case.

Reply:
@nsathyaseelan @kmova -- I have set the openebs/linux-utils image tag as 'latest'. openebs/node-disk-manager uses ndm-operator:ci with linux-utils:ci
The intention was to be able to test provisioner-localpv:ci against a stable release.
Will this interfere with E2e tests?

Reply:
niladrih Ideally image changing these image in the test case based on the regex expression. will modify those tasks it won't affect. Will include all the changes done in the operator file into e2e test.

Reply:
Got it @nsathyaseelan. Thanks.

Reply:
I have changed the linux-utils image tag to `ci`.

Reply:
@kmova -- The E2e changes will come after this PR is merged.
cc: @nsathyaseelan